docs/src/pages/demos/tables/SpanningTable.js

```diff
@@ -29,20 +29,20 @@ function priceRow(qty, unit) {
   return qty * unit;
 }
 
-function createRow(id, desc, qty, unit) {
+function createRow(desc, qty, unit) {
   const price = priceRow(qty, unit);
-  return { id, desc, qty, unit, price };
+  return { desc, qty, unit, price };
 }
 
 function subtotal(items) {
   return items.map(({ price }) => price).reduce((sum, i) => sum + i, 0);
 }
 
 const rows = [
-  ['Paperclips (Box)', 100, 1.15],
-  ['Paper (Case)', 10, 45.99],
```

Comment:
> I don't know what to do with spread operator in `SpanningTable.tsx` because TypeScript considers the spread operator incompatible with the function signature of `createRow`.
> 
> Source:
> [Microsoft/TypeScript#4130](https://github.com/Microsoft/TypeScript/issues/4130)

This was an issue with type widening. TypeScript inferred a nested array `(string | number)[][]` instead of an array of tuples `[string, number, number][]`. With TypeScript 3.4 we could've written `['Paper (Case)', 10, 45.99] as const` to fix this issue.